Replaced include of afxres.h

thekovic opened the pull request:
in Windows Microsoft Visual resource script (removes dependency on Microsoft Foundation Class library for building with Visual Studio)

atsb commented:
Jumping is still there but mouse 2 binding for it is fine I think.  Yep, space is use..  says Doom!

thekovic committed:
Replaced include of afxres.h

in Windows Microsoft Visual resource script (removes dependency on Microsoft Foundation Class library)
